packages/react-storage/src/components/StorageManager/types.ts

```diff
@@ -12,6 +12,15 @@ import {
 } from './ui';
 import { StorageManagerDisplayText, PathCallback, UploadTask } from './utils';
 
+interface BucketInfo {
```

Comment:
Nit: should we share this type between components?

Reply:
Yes, that makes sense to me. It didn't seem substantial enough to create a new shared types file for storage components, so I went with the definition in the Storage Manager types. There are already exported types and interfaces in that file, and none aside from props in Storage Image types.

Reply:
Can we just import and extend `FileUploaderProps` in the assignment of `StorageManagerProps` and dedupe this file?

Reply:
Are you suggesting the removal of this file and providing its exports from FileUploader types? Or reducing `StorageManagerProps` to `export interface StorageManagerProps extends FileUploaderProps {}`?

Reply:
Keeping this file and doing the latter

Reply:
Got it. Thanks for the clarification

Reply:
Updated file to extend `FileUploaderProps` instead of duplicating type declaration